partner_identification_sale/README.rst

```diff
@@ -0,0 +1,74 @@
+===========================
+Partner Identification Sale
+===========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fpartner--contact-lightgray.png?logo=github
+    :target: https://github.com/OCA/partner-contact/tree/14.0/partner_identification_sale
+    :alt: OCA/partner-contact
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/partner-contact-14-0/partner-contact-14-0-partner_identification_sale
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/134/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Adds Partner ID requirements to products with the option to prevent confirming sales orders if there is no valid ID.
```

Comment:
@Chanakya-OSI From your description, I think pulling the stock module is a little bit too much no  ?

Reply:
@rousseldenis the stock module adds the page/sections to the product so this module would add to that which is why it's in the depends. Future modules for manufacturing and such would do the same.

Reply:
I understand, what I wanted to says, is this module that is named ..._sale should depend on stock ?

As Odoo did a sale_stock glue module, shouldn't we do the same and have a partner_identification_sale_stock?

We can imagine someone that has only sale installed and want to restrict sale confirmation with Ids.

Reply:
@rousseldenis I understand what you are saying now and agree this is a better approach. @Chanakya-OSI lets move forward with this suggestion. I think it would just be a matter of removing the stock dependency and changing the product views to be similar to the stock module where if you install this alone then the options would be on the products. Then create a new module called product.template.attribute.value changing the view so they coincide together. Please reach out to me if you have questions.